test/pageobjects/HomePage.js

```diff
@@ -2,38 +2,72 @@ import Base from './Base'
 
 class HomePage extends Base {
     get ToolsQaImg() { return $('img') }
-    get h5() { return $('h5')}
-    get ElementsEntity() { return $('#app > div > div > div.home-body > div > div:nth-child(1)') }
-    get FormsEntity() { return $('#app > div > div > div.home-body > div > div:nth-child(2)') }
-    get AFWEntity() { return $('#app > div > div > div.home-body > div > div:nth-child(3)') } 
-    get WidgetsEntity() { return $('#app > div > div > div.home-body > div > div:nth-child(4)') }
-    get InteractionsEntity() { return $('#app > div > div > div.home-body > div > div:nth-child(5)') }
-    get BookStoreAppEntity() { return $('#app > div > div > div.home-body > div > div:nth-child(6)') }
-    get MainHeader() { return  $('.main-header')}
+
+    get h5_Elements() { return $('h5=Elements') }
+    get h5_Forms() { return $('h5=Forms') }
+    get h5_AFW() { return $('h5=Alerts, Frame & Windows') }
+    get h5_Widgets() { return $('h5=Widgets') }
+    get h5_Interactions() { return $('h5=Interactions')}
+    get h5_BookStoreApp() { return $('h5=Book Store Application')}
+
+    get MainHeader1() { return $('.main-header=Elements')}
+    get MainHeader2() { return $('.main-header=Forms')}
+    get MainHeader3() { return $('.main-header=Alerts, Frame & Windows')}
+    get MainHeader4() { return $('.main-header=Widgets')}
+    get MainHeader5() { return $('.main-header=Interactions')}
+    get MainHeader6() { return $('.main-header=Book Store')}
+
     ToolsQaImgIsVisible(){ 
         this.ToolsQaImg.waitForExist()
```

Comment:
waitForExist() function does no wait for an element to be visible on the page, it just waits for it to be present within the DOM.
The name of the function should match its functionality, in this case the name should say `toolsQaImgExists()`, or the function body should contain the `waitForVisible()` method.